src/transformers/models/esm/configuration_esm.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+# Copyright Facebook and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" ESM model configuration """
+
+from ...utils import logging
+from ..bert.configuration_bert import BertConfig
+
+
+logger = logging.get_logger(__name__)
+
+ESM_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/esm1b": "https://huggingface.co/facebook/esm1b/resolve/main/config.json",
+    # See all ESM models at https://huggingface.co/models?filter=esm
+}
+
+
+class ESMConfig(BertConfig):
```

Comment:
Can you just copy and paste the code instead of subclassing? We would really like models to be as independent as possible.